lib/matplotlib/tests/test_font_manager.py

```diff
@@ -67,11 +67,11 @@ def test_otf():
     if os.path.exists(fname):
         assert is_opentype_cff_font(fname)
 
-    otf_files = [f for f in fontManager.ttffiles if 'otf' in f]
-    for f in otf_files:
-        with open(f, 'rb') as fd:
-            res = fd.read(4) == b'OTTO'
-        assert res == is_opentype_cff_font(f)
+    for f in fontManager.ttflist:
+        if 'otf' in f.name:
+            with open(f.name, 'rb') as fd:
```

Comment:
This is not hit when the tests are run.  Maybe we have to otf fonts on travis?

Reply:
As discussed on this week's call, this is actually a real drop in coverage due to a typo (`f.name` vs. `f.fname`)